tgui/packages/tgui/interfaces/AntagInfoDemon.tsx

```diff
@@ -0,0 +1,136 @@
+import { useBackend } from '../backend';
+import { Box, Section, Stack } from '../components';
+import { BooleanLike } from 'common/react';
+import { Window } from '../layouts';
+
+const jauntstyle = {
+  color: 'lightblue',
+};
+
+const injurestyle = {
+  color: 'yellow',
+};
+
+type Objective = {
+  count: number;
+  name: string;
+  explanation: string;
+  complete: BooleanLike;
+  was_uncompleted: BooleanLike;
+  reward: number;
+}
+
+type Info = {
+  fluff: string;
+  explain_attack: BooleanLike;
+  objectives: Objective[];
+};
+
+export const AntagInfoDemon = (props, context) => {
+  const { data } = useBackend<Info>(context);
+  const {
+    fluff,
+    objectives,
+    explain_attack,
+  } = data;
+  return (
+    <Window
+      width={620}
+      height={356}
+      theme="syndicate">
+      <Window.Content
+        style={{ 'background-image': 'none' }}>
+        <Stack fill>
+          <Stack.Item>
+            <DemonRunes />
+          </Stack.Item>
+          <Stack.Item grow>
+            <Stack vertical width="544px" fill>
```

Comment:
why hardcode the width here? What if someone decides to resize the UI?

Reply:
It gets farked. It's sadly an IE bug i'm pretty sure that forced me to do this